api/include/mfxstructures.h

```diff
@@ -124,6 +124,9 @@ enum {
     MFX_FOURCC_Y216         = MFX_MAKEFOURCC('Y','2','1','6'),
     MFX_FOURCC_Y416         = MFX_MAKEFOURCC('Y','4','1','6'),
 #endif
+#if (MFX_VERSION >= MFX_VERSION_NEXT)
+    MFX_EXTBUFF_PARTIAL_BITSTREAM_PARAM         = MFX_MAKEFOURCC('P','B','O','P'),
```

Comment:
Why do you place extbuf param in the list of FOURCC color formats? If it is color format - name it accordingly. If it is not - place elsewhere.